packages/react-devtools-extensions/package.json

```diff
@@ -62,10 +62,10 @@
     "source-map-js": "^0.6.2",
     "sourcemap-codec": "^1.4.8",
     "style-loader": "^0.23.1",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3",
-    "workerize-loader": "^1.3.0"
+    "webpack": "^5",
+    "webpack-cli": "^4",
+    "webpack-dev-server": "^4",
+    "workerize-loader-5": "^1.4.0"
```

Comment:
Something about this change causes the following error to be thrown in the extension process:

```
Uncaught ReferenceError: process is not defined
```

Looking at the code in question, I see:
```js
(0, _utils.default)("ArrayExpression", {
  fields: {
    elements: {
      validate: (0, _utils.chain)((0, _utils.assertValueType)("array"), (0, _utils.assertEach)((0, _utils.assertNodeOrValueType)("null", "Expression", "SpreadElement"))),
      default: !process.env.BABEL_TYPES_8_BREAKING ? [] : undefined
    }
  },
  visitor: ["elements"],
  aliases: ["Expression"]
});
```

Seems like this is coming from an imported module?

Seems like the result of this error is that the named hooks feature does not work. (Clicking the little "🪄" button for an inspected element eventually times out whereas it used to work.)

Reply:
will look into this further

Reply:
🙇🏼‍♂️ Thanks!